pocs/camera/camera.py

```diff
@@ -302,7 +302,13 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
 
         # pop exptime from kwarg as its now in exptime
         exptime = kwargs.pop('exptime', observation.exptime.value)
-        exposure_event = self.take_exposure(seconds=exptime, filename=file_path, **kwargs)
+
+        # Move the filerwheel into position
+        if (self.filterwheel is not None) and (observation.filter_name is not None):
+            self.filterwheel.move_to(observation.filter_name)
```

Comment:
```suggestion
            self.filterwheel.move_to(observation.filter_name, blocking=True)
```

Need to either directly block or capture the returned event and check it. Don't want to start an exposure in the middle of a filter movement.

Reply:
Seems like we also need to do some error checking here in case movement fails.